scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from numpy.testing import assert_array_almost_equal
+from scipy.spatial.transform import Rotation
+
+
+def test_generic_quat_matrix():
+    x = np.array([[3,4,0,0], [5, 12, 0, 0]])
+    r = Rotation.from_quaternion(x)
+    expected_quat = x / np.array([[5], [13]])
+    assert_array_almost_equal(r._quat, expected_quat)
+
```

Comment:
I believe PEP8 prescribes to use 2 blank lines between standalone functions and classes (and one between methods in classes). It is just sort of a convention.